iRecruit-ui/src/components/Main/Main.jsx

```diff
@@ -7,7 +8,7 @@ import Sidebar from "../Sidebar/Sidebar";
 import Home from "../Home/Home";
 import About from "../About/About";
 import Contact from "../Contact/Contact";
-import { Routes, Route, useLocation} from "react-router-dom";
+import { BrowserRouter, Routes, Route, useLocation} from "react-router-dom";
```

Comment:
I think you need to wrap `Routes` with `BrowserRouter`

Reply:
I tried that but it returns an error saying I can't use BrowserRouter in more than one component.It was in my App.jsx, which contains my Main.jsx.

<img width="583" alt="image" src="https://github.com/Henrywis/Capstone-project/assets/105118133/5708fa01-4afe-43ba-8283-efc0d0c6cf6d">

Reply:
![image](https://github.com/Henrywis/Capstone-project/assets/105118133/29e8dd25-6873-4409-a998-215a1b6b4ec2)



Resolved:

In this code snippet, the wildcard route with path="/*" is used as a catch-all route and checks the user variable to decide what to render, which means that it will match any URL that hasn't been matched by the previous defined routes.
In the previous code snippet, the path="/" route is set as the default route and doesn't have a condition based on the user variable.